src/DynamicData/Cache/Internal/TransformByValues.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) 2011-2023 Roland Pheasant. All rights reserved.
+// Roland Pheasant licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for full license information.
+
+using System.Reactive;
+using System.Reactive.Linq;
+
+namespace DynamicData.Cache.Internal;
+
+internal sealed class TransformByValues<TDestination, TSource, TKey>
```

Comment:
Is this the same a Convert?

See https://github.com/reactivemarbles/DynamicData/blob/main/src/DynamicData/Cache/ObservableCacheEx.cs#L1014.


Maybe that could be resurrected?

Reply:
Uhhh, yep, that looks pretty much the same, save for a few technical optimizations.

I think it's worth considering that `.Convert()` has a bit of a discoverability problem, then. So, we could consider options for addressing that:

1. Add `TransformByValues()` (or whatever name we agree on) anyway, and have it and `.Convert()` both call the same implementation behind the scenes.
2. #1 but also mark `.Convert()` as Obsolete, to be removed some day.
3. Just try and address it with some new Wiki content, as Darrin already suggested. In which case, I would still swap `.Convert()` over to the implementation I wrote here.